pages/agent/v3/signed-pipelines.md

```diff
@@ -0,0 +1,114 @@
+# Signed pipelines
+
+## What are signed pipelines?
+
+Signed Pipelines is a security feature on the Buildkite Agent that allows you to cryptographically sign steps that are uploaded Buildkite, and then verify that signature on the Buildkite Agent before running the step. This can prevent a malicious actor from modifying the steps that are run on your agents, especially in the (hopefully unlikely) event that the Buildkite backend is compromised.
+
+These signatures mean that while a threat actor could modify jobs in flight, when the Agent runs those steps, when it tries to verify the signature, it will fail, and the step will not be run.
```

Comment:
```suggestion
These signatures mean that if a threat actor were able to modify a job in flight, the Agent would refuse to run it due to mismatched signatures.
```

Simpler, and changes from "while something bad can happen" to "if it were somehow possible" - slightly different language